WordPress/src/main/java/org/wordpress/android/ui/notifications/services/NotificationsPendingDraftsService.java

```diff
@@ -0,0 +1,229 @@
+            return;
+        }
+        Intent intent = new Intent(context, NotificationsPendingDraftsService.class);
+        context.startService(intent);
+    }
+
+    @Override
+    public IBinder onBind(Intent intent) {
+        return null;
+    }
+
+    @Override
+    public void onCreate() {
+        super.onCreate();
+        AppLog.i(AppLog.T.NOTIFS, "notifications pending drafts service > created");
+    }
+
+    @Override
+    public void onDestroy() {
+        AppLog.i(AppLog.T.NOTIFS, "notifications pending drafts service > destroyed");
+        super.onDestroy();
+    }
+
+    @Override
+    public int onStartCommand(Intent intent, int flags, int startId) {
+        if (intent != null) {
+            performDraftsCheck();
+        }
+        return START_NOT_STICKY;
+    }
+
+    private void performDraftsCheck() {
+        if (running) {
+            return;
+        }
+        running = true;
+        /*
+        1) check all “local” drafts, and check that they have been pending for more than 3 days.
+        2) make notification if ONE draft and if more than ONE make another text
+        ONE: “You’ve got this draft pending for publishing for xxx days. Would you like to check it?”
+        (we also have a generic message if we can't determine for how long a draft has been sitting there or it's been
+        more than 30 days)
+        MORE: “You’ve got 2 posts in drafts. Want to check them?”
+        * */
+        new Thread(new Runnable() {
+            @Override
+            public void run() {
+                ArrayList<Post> draftPosts =  WordPress.wpDB.getDraftPostList(WordPress.getCurrentBlog().getLocalTableBlogId());
+                ArrayList<Post> draftPostsOlderThan3Days = new ArrayList<>();
+                if (draftPosts != null && draftPosts.size() > 0) {
+                    //now check those that have been sitting there for more than 3 days now.
+                    long now = System.currentTimeMillis();
+                    long three_days_ago = now - (ONE_DAY * 3);
+                    for (Post post : draftPosts) {
+                        if (post.getDateLastUpdated() < three_days_ago) {
+                            draftPostsOlderThan3Days.add(post);
+                        }
+                    }
+
+                    //check the size and build the notification accordingly
+                    long daysInDraft = 0;
+                    if (draftPostsOlderThan3Days.size() == 1) {
+                        Post post = draftPostsOlderThan3Days.get(0);
+                        daysInDraft = (now - post.getDateLastUpdated()) / ONE_DAY;
+                        long postId = post.getLocalTablePostId();
+                        boolean isPage = post.isPage();
+
+                        //only notify the user if the last time they have been notified about this particular post was at least
+                        //30 minutes ago, but let's not do it constantly each time the app comes to the foreground
```

Comment:
Comment is outdated. It refers to 30 minutes instead we changed it to 24h. Better to put the constant name there?